### PR TITLE
feat: automatically display changelog after update

### LINKED ISSUE #119: [REQUEST] Display installer changelog after update
**Is your feature request related to a problem? Please describe.**
No

**Describe the feature you'd like to see implemented**
Display the installer changelog window after it got updated

**Additional context**
#117 window already exists, code to enable displaying:

``
function showchangelog() {
    const showchangelog = true;
    store.dispatch({ type: 'CHANGELOG', payload: {
        showchangelog
    } });
}
``

All it lasts is to find out how to trigger that properly

Discord username (if different from GitHub):
Foxtrot Sierra#6420

### PR DESCRIPTION
**Pull request by FoxtrotSierra6829**
Fixes #119 

## Summary of Changes
automatically display changelog after update

## Screenshots (if necessary)
grafik

## Additional context

Discord username (if different from GitHub):
Foxtrot Sierra#6420

### CODE REVIEW

File: src/renderer/components/ChangelogModal/index.tsx [hunk 1]
```diff
@@ -33,6 +37,15 @@ function hidechangelog() {
         showchangelog
     } });
 }
+function versionchanged() {
```
Comment (ZigTag) [inline src/renderer/components/ChangelogModal/index.tsx:40]: `versionChanged`, also it should be a const function component `const versionChanged = () => {`

**Review by ZigTag — CHANGES_REQUESTED** [review]

**Review by ZigTag — APPROVED** [review]
LGTM

**Review by ZigTag — COMMENTED** [review]
Lgtm

**Review by ZigTag — APPROVED** [review]
Lgtm

File: src/main/index.ts [hunk 2]
```diff
@@ -145,6 +145,7 @@ app.on('second-instance', () => {
 // code. You can also put them in separate files and import them here.
 
 function configureSettings(app: App) {
+
```
Comment (nistei) [inline src/main/index.ts:148]:
```suggestion
```

**Review by nistei — CHANGES_REQUESTED** [review]

**Review by nistei — APPROVED** [review]
LGTM

**Pull request merged**